source/whats-new.txt

```diff
@@ -12,3 +12,250 @@ What's New
    :depth: 1
    :class: singlecol
 
+Learn what's new in:
+
+* :ref:`Version 1.10 <version-1.10>`
+* :ref:`Version 1.9 <version-1.9>`
+* :ref:`Version 1.8 <version-1.8>`
+* :ref:`Version 1.7 <version-1.7>`
+* :ref:`Version 1.6 <version-1.6>`
+* :ref:`Version 1.5 <version-1.5>`
+* :ref:`Version 1.4 <version-1.4>`
+* :ref:`Version 1.3 <version-1.3>`
+* :ref:`Version 1.2 <version-1.2>`
+* :ref:`Version 1.1 <version-1.1>`
+* :ref:`Version 1.0 <version-1.0>`
+
+
+.. _version-1.10.0:
+.. _version-1.10:
+
+What's New in 1.10
+--------------------
+
+.. important:: Upgrade to Version 1.10.1 or Higher 
+
+  The 1.10.1 Go driver patches a bug that can cause data corruption when 
+  rotating :ref:`Data Encryption Keys <csfle-key-architecture>` (DEKs) encrypted 
+  with a :ref:`Customer Master Key <csfle-key-architecture>` hosted on Google 
+  Cloud Key Management Service or Azure Key Vault.
+
+New features of the 1.10 Go driver release include: 
+
+- Added full support for MongoDB 6.0.
+  
+- Support for new features related to :ref:`qe-manual-feature-qe`,
+  including new options for automatic and manual encryption.
+
+- Support for the new Automatic Encryption Shared Library, which replaces the 
+  ``mongocryptd`` process. The shared library requires MongoDB v6.0 Enterprise 
+  or later and ``libmongocrypt`` 1.5.0 or later.
+
+- Added :ref:`clustered index <golang-clustered-indexes>` creation support.
+
+- Added a new API and ``ClientEncryption`` entity operations for encryption key 
+  management.
+
+- Added a ``Timeout`` client option to set a default context timeout for 
+  each operation sent through that client.
+
+- Implemented a patch to default data to either ``primitive.M`` or 
+  ``primitive.D`` when decoding empty types.
+
+- Support for encoding atypical map key types for data that can be unmarshalled 
+  into a textual representation of itself. 
+
+- Performance optimizations including:
+  
+  - Improved full document requests for before and after updates in change 
+    stream events.
+
+  - Improved PRN and UUID generation 
+
+  - Reduced memory consumption when compressing wire messages.
+
+- Added troubleshooting support for `frequently encountered issues 
+  <https://github.com/mongodb/mongo-go-driver/blob/master/docs/common-issues.md>`__
+
+
+.. _version-1.9:
+
+What's New in 1.9
+-------------------
+
+New features of the 1.9 Go driver release include: 
+
+- Improved connection storm mitigation.
+
+- Added ``Custom```` options to aggregate and change stream operations.
+
+- Added a ``Let`` option to most CRUD commands that specifies parameters for use 
+  in an aggregate expression context. ``Let`` must be a document that maps 
```

Comment:
Issue: [Ambiguous compound modifier.
](https://www.mongodb.com/docs/meta/style-guide/style/punctuation/#hyphens-in-compound-modifiers) This could be interpreted as either "an expression context that has been aggregated" or "a context of type aggregate expression".

Suggestion (assuming the context is of type "aggregate expression"):
```suggestion
  in an aggregate-expression context. ``Let`` must be a document that maps 
```

Reply:
Opting to just take out the word "context" since its removal doesn't seem to change the meaning of the statement and mitigates this confusion.